tbx/core/blocks.py

```diff
@@ -302,3 +302,72 @@ class StoryBlock(StreamBlock):
 
     class Meta:
         template = "patterns/molecules/streamfield/stream_block.html"
+
+
+class ServiceStoryBlock(StreamBlock):
+    h2 = CharBlock(
```

Comment:
I think this might be able to extend StoryBlock and not repeat the definitions here - @engineervix will know better than me. The downside of doing that would be if we ever add any further blocks to the StoryBlock in the future that shouldn't be on the Service page it will be harder to sort out. Any thoughts @engineervix ?

Reply:
Yes, I was also thinking this but didn't do it for that reason

Reply:
I agree that it's better to just extend the the StoryBlock, since we are, at least at this point, including everything there. This way, maintenance is easier because if there's a bug or change on one of the blocks, we don't have to make changes in multiple places.